src/Base/FileInfo.h

```diff
@@ -34,6 +34,14 @@
 namespace Base
 {
 
+/// When reading and writing a character stream, the incoming data can be dumped into the stream

```

Comment:
Putting this enum to FileInfo.h seems wrong to me as it's used in the Reader class.